Dockerfile

```diff
@@ -49,30 +40,27 @@ RUN \
     /tmp/sabnzbd.tar.gz -C \
     /app/sabnzbd --strip-components=1 && \
   cd /app/sabnzbd && \
-  pip3 install -U pip && \
-  pip install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/ubuntu/ \
+  python3 -m pip install --upgrade pip && \
+  pip3 install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/alpine/ \
+    wheel \
     apprise \
     pynzb \
     requests && \
-  pip install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/ubuntu/ -r requirements.txt && \
+  pip3 install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/alpine/ -r requirements.txt && \
```

Comment:
for 3.15, we should use `--find-links https://wheel-index.linuxserver.io/alpine-3.15/`

Reply:
Ah, forgot about that change. Will fix.